tile/targets/cpu/llvm_cpu.jsonnet

```diff
@@ -172,7 +171,7 @@ local PARAMS = {
             },
 
             // Init aggregation outputs
-            // Keet this towards the end since other passes are generating intermediate blocks and the initialization 
+            // Keet this towards the end since other passes are generating intermediate blocks and the initialization
```

Comment:
Typo: Keet -> Keep?

Reply:
This is the old wording. We will anyway replace this section soon with the new agg_init pass soon.